jsapp/js/components/formSummaryProjectInfo.tsx

```diff
@@ -73,7 +73,9 @@ export default function FormSummaryProjectInfo(
               <bem.FormView__label>
                 {metadata.description?.label ?? t('Description')}
               </bem.FormView__label>
-              {props.asset.settings.description || '-'}
+              <div dir='auto'>
+                {props.asset.settings.description || '-'}
+              </div>
```

Comment:
This doesn't seem to be working, the description is staying left aligned.
![Screenshot 2024-07-30 at 1 04 42 PM](https://github.com/user-attachments/assets/d0ef441e-6e30-46ea-8aa0-f27acb0272ee)